### PR TITLE
Issue #62 resolved.

### PR DESCRIPTION
**Pull request by pranitb10**
This contains the solution for Issue #62 i.e Implement Container and Bootstrap grid throughout the page. Also, I have removed the extra unnecessary inline properties.

### CODE REVIEW

**Review by Amorpheuz — CHANGES_REQUESTED** [review]
I don't think there is a need for multiple containers throughout the site, all the containers can be replaced by a single container containing all the rows. Also, the navbar shouldn't be part of a container. You might want to go through [this](https://getbootstrap.com/docs/4.4/layout/overview/) and [this](https://uxplanet.org/how-the-bootstrap-4-grid-works-a1b04703a3b7) for some more info!

**Comment by pranitb10** [comment]
Sure @Amorpheuz . On it!

**Comment by arps18** [comment]
@Amorpheuz I've solved conflict in the file, so kindly review if any changes are to be done!

File: style/style.css [hunk 1]
```diff
@@ -305,7 +328,7 @@ html{
   
   .accordion-content {
     padding: 0 20px;
-    margin-left: 100px;
+    margin-left: 40px;
```
Comment (Amorpheuz) [inline style/style.css]: Don't edit things not related to your issue xD, they might be getting fixed in another issue.

File: index.html [hunk 2]
```diff
@@ -243,220 +305,211 @@
...
                             <a href="" target="_blank">
                             <div class="card4" style="width: 250px; height: 210px;">
-                            <p style="margin-left:10px;"><img class="img" src="./assets/mozguj-horizontal-black.png"
-                                    alt="user" height="100px" width="230px"></p>
-                            <p style="font-size: 30px; margin-left: 8px;"><b>MozPU</b><br><span
-                                    style="font-size: 25px;">PU</span></p>
+                                <p style="margin-left:10px;"><img class="img" src="./assets/mozguj-horizontal-black.png"
+                                        alt="user" height="100px" width="230px"></p>
+                                <p style="font-size: 30px; margin-left: 8px;"><b>MozNUV</b><br><span
+                                        style="font-size: 25px;">NUV</span></p>
```
Comment (Amorpheuz) [inline index.html:361]: Why the change from PU to NUV?

Comment (pranitb10) [reply index.html:361]: Oops, sorry might have done that by mistake because in my local I had two PU and no NUV so that's why. Sorry for that.

File: style/style.css (hunk 1 above)
Comment (pranitb10) [reply style/style.css]: Okay, have set it back to as it was before.
